src/index.js

```diff
@@ -115,6 +116,8 @@ async function run(params) {
     params.__ow_query = query;
   }
   const url = `${params.__ow_path.substring(1)}?${params.__ow_query || ''}`;
```

Comment:
Maybe we should use the `url-parse` module here to split out the hostname? It's already in the dependency tree and would feel safer...

Reply:
Or just `url`

Reply:
or we pass the `url` as separate param, like mentioned in #187